docs/plugins/purchase.md

```diff
@@ -88,72 +84,32 @@ Purchase.setup('http://csp-backend.com/api', {
     uri: '/subscriptions/:id',
     method: 'DELETE',
   },
-  signature: {
-    uri: '/assets/:id/signature',
-    method: 'POST',
-    headers: {
-      'x-auth-token': 'abcd123xyz'
-      'cache-control': 'no-cache'
-    }
-  },
 })
 ```
 
-In some cases the CSP backend setup is too different from the default specification, that it becomes too difficult
-to configure the enpoint uri's. In those cases you can specify a `callback` instead, and handle the request to the
-CSP backend entirely yourself.
+> If the CSP backend does *not* completely follow the expected blueprint, or can not be configured properly via the `setup` method, the steps in the `buy` method can also be implemented *manually*. In that case, you can use the Plugin methods `signature`, `payment`, `subscribe` and `confirm` (see [below](#signature)).
 
-The callback function will receive 2 arguments: `params` and `data` and should _always_ return a `Promise` which resolves the result (or rejects an error in case of a failure). When a callback is specified, all other keys in that `endpoint` config
-are ignored.
+### assets
 
+The `assets` method returns a *Promise* with the result of the request that is made to the CSP to retrieve available assets for purchase. Typically, it returns an Array of objects containing information about the assets (such as *title*, *description* and *price*).
 
-```js
-Purchase.setup('http://csp-backend.com/api', {
-  asset: {
-    callback(params, data) {
-      return Promise((resolve, reject) => {
-        Api.getAssetDetails(params.id)
-          .then(resolve)
-          .catch(reject)
-      }
-    }
-  },
-  signature: {
-    callback(params, data) {
-      return fetch('http://csp-backend.com/api/' + data.household + '?productId=' + params.id, {
-        headers: {
-          'Accept': 'application/json',
-				  'Content-Type': 'application/json'
-        }),
-        method: 'POST',
-      })
-    }
-  },
-})
-```
-
-### Assets
-
-The `assets` method returns a Promise with the result of the request to the CSP to retrieve assets available for purchase. In most cases it would return an `Array` of objects with information about the assets (i.e. _title_, _description_ and _price_).
-
-The `household` is automatically sent as a `query param` in the request, allowing to optionally return information on whether the asset has been _purchased_ by the current household.
-
-The `assets` method is an _optional_ convenience method exposed by the Purchase plugin. Depending on the CSPs API integration, the functionality to retrieve assets can be implemented fully custom in an App.
+You can use the `household` option to return information on whether the assets were purchased by the current household or not. It is automatically added as a `query param` in the request.
```

Comment:
I think the previous sentence was more clear ..

Reply:
I have rewritten the sentence as: "The `household` is automatically sent as a `query param` in the request, which allows to optionally return information on whether the assets were purchased by the current household or not."